### PR TITLE
aredn: Prepare 3.19.3.0 release

### PR DESCRIPTION
**Pull request by ae6xe**

### CODE REVIEW

**Comment by K6AH** [comment]
Joe, check spelling on line  231, "AREDN releaes 3.19.3.0 "

File: README.md [hunk 1]
```diff
@@ -218,6 +228,12 @@ You need approximately 10GB of space for the build.
 Prior AREDN images can be rebuilt.  Replace one of the following after
 the "cd aredn_ar71xx" command above:
 
+AREDN releaes 3.19.3.0
```
Comment (dman776) [inline README.md]: typo;   (releases)

Comment (ae6xe) [reply README.md]: Typo has been resolved.

**Review by dman776 — APPROVED** [review]
looks good

**Pull request merged**